src/service/PetClinicService.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+  const URL:string = 'http://localhost:9966/petclinic/api';
```

Comment:
(Optional) I don't like writing a URL in code. Consider using an .env file please.

Reply:
I had a look but there seems to be multiple ways to create env files. 
Are you referring to this? 
https://stackoverflow.com/questions/49579028/adding-an-env-file-to-a-react-project#answer-49579700

Reply:
Vite provides some good approaches to use env files/variables.